be/src/exec/partition/chunks_partitioner.h

```diff
@@ -157,23 +177,24 @@ class ChunksPartitioner {
 
             while (chunk_it != chunk_end) {
                 if (!consumer(partition_idx, *chunk_it++)) {
```

Comment:
here `*chunk_it++`, suggest to use `std::move(*chunk_it++)`, once the chunk is pulled from the PartitionChunk, PartitionChunk need not hold it any more,  so the memory will be released ASAP,  especially in drastic data skew cases.

Reply:
Good point, I'll update.